### PR TITLE
Fixing Missing module in requirements.txt #99

### PR DESCRIPTION
**Pull request by Swarno-Coder**
This PR enabling to fix the BUG pointing on the ISSUE #99.
It adds scipy -> requirements.txt file, that fixes the ISSUE & BUG.

### CODE REVIEW

File: requirements.txt
Comment (Swarno-Coder) [inline requirements.txt:1]: While running the .\visualizer_drag.py it gives the error:
Traceback (most recent call last):
  File "D:\codes\LangChatAI\DragGAN\visualizer_drag.py", line 17, in <module>
    from gui_utils import imgui_window
  File "D:\codes\LangChatAI\DragGAN\gui_utils\imgui_window.py", line 15, in <module>
    from . import text_utils
  File "D:\codes\LangChatAI\DragGAN\gui_utils\text_utils.py", line 16, in <module>
    import scipy.ndimage
ModuleNotFoundError: No module named 'scipy'
We can fix this error by adding scipy in requirements.txt file

File: requirements.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ hf_transfer
 pyopengl
 imgui
 glfw
+scipy
```
Comment (Swarno-Coder) [inline requirements.txt:10]: Scipy module name added to install while installing other modules

**Comment by XingangPan** [comment]
Thanks for the PR. Another PR has fixed this.

**Comment by Swarno-Coder** [comment]
@XingangPan Thanks sir for this

**Pull request closed without merging**